tests/onnx/data/reference_graphs/quantization/synthetic/gemm_weight_transpose_model.dot

```diff
@@ -0,0 +1,19 @@
+strict digraph  {
+"0 QuantizeLinear_X_1" [id=0, type=QuantizeLinear];
+"1 DequantizeLinear_X_1" [id=1, type=DequantizeLinear];
+"2 Identity" [id=2, type=Identity];
+"3 QuantizeLinear_W_1" [id=3, type=QuantizeLinear];
+"4 DequantizeLinear_W_1" [id=4, type=DequantizeLinear];
+"5 Gemm" [id=5, type=Gemm];
+"6 Softmax" [id=6, type=Softmax];
+"7 nncf_model_input_0" [id=7, type=nncf_model_input];
+"8 nncf_model_output_0" [id=8, type=nncf_model_output];
+"0 QuantizeLinear_X_1" -> "1 DequantizeLinear_X_1"  [label="[1, 10]", style=dashed];
+"1 DequantizeLinear_X_1" -> "2 Identity"  [label="[1, 10]", style=solid];
```

Comment:
Is the `Identity -> Gemm` the combined node? I mean, why do we place the `QuantizeLinear + DequantizeLinear` pair before it, not after?

Reply:
We place QDQ on the edge where the weight is. This is because of propagation till the weigth tensor

Reply:
Here the weight is an input of `Identity`

Reply:
Not sure that it will correctly work for the `Reshape` layer, for example.
IMHO, it's better when we put the `QDQ` pair right before the `Convolution`/`MatMul`/etc.

Reply:
Reshape layers are a special case when we return output. Please, take a look at the examples in the tests.

Reply:
I am not sure whether we can apply weight compression to int8 in your case